src/pybind/mgr/prometheus/module.py

```diff
@@ -516,7 +517,8 @@ def _get_value(message, delim=' ', word_pos=0):
                         v, err = _get_value(message)
 
                     if err:
-                        self.log.error("healthcheck {} message format is incompatible and has been dropped".format(check.name))
+                        self.log.error(
+                            "healthcheck {} message format is incompatible and has been dropped".format(check.name))
```

Comment:
Nit: `logging` supports [lazy interpolation with `str.format()`](https://docs.python.org/3/library/logging.html#logging.Formatter)

Reply:
thanks for your suggestion, updated accordingly.